### ISSUE TITLE
Native Knobs: Knob menu showing on emulator screen but not in StoryBook web UI 

### ISSUE THREAD

**Issue by feluxe** [storybookjs/react-native#51]
**Describe the bug**

I use StoryBook with React Native which works nicely. I can use the StoryBook web UI in the browser on my laptop to select different Stories on my device emulator. I was hoping to achieve the same with the `Knobs` addon, but it doesn't really work like I expected: The knobs are only availabe on my emulator screen, but not on the web UI in my browser.

**To Reproduce**

* Install `@storybook/addon-knobs`
* Install `@storybook/addon-ondevice-knobs`
* Create a knob.
* I can now find a working knob-menu on my emulator screen.
* But I can't find a knob-menu on the web UI in my Browser.

**Expected behavior**

I was expecting to find a knob-menu in the StoryBook web UI as well.

**System:**
```
Environment Info:

  System:
    OS: Linux 5.6 Arch Linux
    CPU: (8) x64 Intel(R) Core(TM) i7-8550U CPU @ 1.80GHz
  Binaries:
    Node: 12.16.2 - ~/.asdf/installs/nodejs/12.16.2/bin/node
    Yarn: 1.22.4 - /usr/bin/yarn
    npm: 6.14.4 - ~/.asdf/installs/nodejs/12.16.2/bin/npm
  Browsers:
    Firefox: 76.0

```

**Comment by SleepingWhale** [comment]
I also would love to have this capability. But according to [Addon / Framework Support Table](https://github.com/storybookjs/storybook/blob/master/ADDONS_SUPPORT.md) knob is available as on device addon only.

**Comment by IShotTheSheriff** [comment]
It's the same on my side. It's like content of `addons.js` is ignored for some reason.

**Comment by Esxiel** [comment]
Perhaps rather than just the knobs, you're missing all the addons? If so, you guys would probably benefit from referring to #24 . I managed to make my addons(including knobs) show up in the web UI.

**Comment by dannyhw** [comment]
were you able to get things working?

**Comment by dannyhw** [comment]
resolved by #100
